tests/unit-pass/global-constexpr.C

```diff
@@ -0,0 +1,3 @@
+constexpr int a = 1;
+constexpr int b = ::a;
```

Comment:
this test isn't sufficient to exercise this code because it should have already been passing, just with b being dynamically initialized at program startup. I would recommend the following pattern for testing that constexpr is working as intended:
```
#include<stdlib.h>

constexpr int a = 1;
const extern int b;
int c = b;
constexpr int b = a;

int main() {
  if (c != 1) abort();
}
```

Here the test passes if and only if c is initialized after b, which occurs only if b is initialized statically and c is initialized dynamically.